lib/blocs/otp_bloc.dart

```diff
@@ -6,8 +6,10 @@ import 'package:wastexchange_mobile/models/otp_data.dart';
 import 'package:wastexchange_mobile/models/otp_response.dart';
 import 'package:wastexchange_mobile/resources/user_repository.dart';
 import 'package:wastexchange_mobile/util/constants.dart';
+import 'package:wastexchange_mobile/util/logger.dart';
 
 class OtpBloc {
+  final logger = getLogger('OtpBloc');
```

Comment:
Is there a way to call loger directly instead of assigning to logger variable here?

Something like that?

Logger.i("do print something");  

This implementation is awesome. Great 👍

Reply:
@chandru612 Thanks for the feedback. Won't be able to get rid of it if we want to print the classname in the logs which is useful. Resolving for now.